src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/organizations.py

```diff
@@ -75,8 +99,7 @@ def enable_organization_policies(
         """
         try:
             self.client.enable_policy_type(
-                RootId=self.get_ou_root_id(),
-                PolicyType=policy_type
+                RootId=self.get_ou_root_id(), PolicyType=policy_type
```

Comment:
Personally I like the one split on multiple lines more.
As this will work for 1 to many new lines if those get added later.
Plus could you add the trailing comma?